src/diagonal.works/b6/api/vm.go

```diff
@@ -23,6 +23,7 @@ type Context struct {
 	FunctionSymbols FunctionSymbols
 	Adaptors        Adaptors
 	Context         context.Context
+	UseGenericFlow  bool
```

Comment:
Good use of feature flags here. Maybe add a comment though saying it's a feature flag, rather than an option you'd normally use, like those above.